src/node_os.cc

```diff
@@ -87,29 +86,16 @@ static void GetOSType(const FunctionCallbackInfo<Value>& args) {
     return args.GetReturnValue().SetUndefined();
   }
 
-  args.GetReturnValue().Set(
-      String::NewFromUtf8(env->isolate(), info.sysname, NewStringType::kNormal)
-          .ToLocalChecked());
-}
-
-
-static void GetOSRelease(const FunctionCallbackInfo<Value>& args) {
-  Environment* env = Environment::GetCurrent(args);
-  uv_utsname_t info;
-  int err = uv_os_uname(&info);
-
-  if (err != 0) {
-    CHECK_GE(args.Length(), 1);
-    env->CollectUVExceptionInfo(args[args.Length() - 1], err, "uv_os_uname");
-    return args.GetReturnValue().SetUndefined();
-  }
+  // [sysname, version, release]
+  std::vector<Local<Value>> osInformation = {
+    FIXED_ONE_BYTE_STRING(env->isolate(), info.sysname),
+    FIXED_ONE_BYTE_STRING(env->isolate(), info.version),
+    FIXED_ONE_BYTE_STRING(env->isolate(), info.release)
+  };
 
-  args.GetReturnValue().Set(
-      String::NewFromUtf8(env->isolate(), info.release, NewStringType::kNormal)
-          .ToLocalChecked());
+  args.GetReturnValue().Set(Array::New(env->isolate(), osInformation.data(), osInformation.size()));
```

Comment:
… this:

```suggestion
  args.GetReturnValue().Set(Array::New(env->isolate(), osInformation, arraysize(osInformation)));
```

gets rid of an unnecessary heap allocation performed by `std::vector` :)

Reply:
Get it, get it, It was a recomendation of a co-worker, building and testing as you recomend me before, thank you ❤️